__tests__/app.test.js

```diff
@@ -17,15 +17,34 @@ describe("/api/topics", () => {
       .get("/api/topics")
       .expect(200)
       .then((response) => {
-        const topics = response.body.topics.rows;
+        const topics = response.body.topics;
         expect(Array.isArray(topics)).toBe(true);
+        expect(topics.length > 0).toBe(true);
         topics.forEach((topic) => {
           expect(topic).toHaveProperty("slug", expect.any(String));
           expect(topic).toHaveProperty("description", expect.any(String));
         });
       });
   });
+});
+
+describe("invalid enpoint", () => {
   test("GET: 404, Responds with a 404 status code due to an invalid endpoint", () => {
     return request(app).get("/api/not-a-route").expect(404);
   });
 });
+
+describe("/api", () => {
+  test("GET: 200, Should: Be available on /api. Document all other endpoints available.", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        console.log(response.body);
+        const endpoints = response.body;
+        expect(endpoints).toHaveProperty("GET /api");
+        expect(endpoints).toHaveProperty("GET /api/topics");
+        expect(endpoints).toHaveProperty("GET /api/articles");
+      });
```

Comment:
You will have to keep updating this for every endpoint, so it's not dynamic. 

Require in the json and use that to compare in the test, then it will be dynamic 👍